src/components/involve/involve.js

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
 import TwoElementOverlapComponent from '../common-components/TwoElementOverlapComponent/TwoElementOverlapComponent'
 import ImageCluster from '../common-components/ImageCluster/ImageCluster'
+import LargeTextBlock from '../common-components/LargeTextBlock/LargeTextBlock'
+import TextBlockButton from '../common-components/TextBlockButton/TextBlockButton'
+import ProjectBlock from '../common-components/ProjectBlock/ProjectBlock'
 import './involve.css';
 
 
 /**
  * @returns {React.Component}
  */
 const GetInvolve = props => {
-    let imageCluster = () => {
-        let frontImage = <div className="involve-image"></div>
-        let backImage = <div className="involve-image"></div>
-        return <TwoElementOverlapComponent childForeground={frontImage} childBackground={backImage}/>
-    }
-
-    let imageCluster2 = () => {
-        let sample1 = <div className="involve-image"></div>
-        let sample2 = <img class="imageEnterAnimate image priority-2 order-3 cluster-design-1" src="../common-components/ImageCluster/sample-images/mountain-1" alt="gudetama-1" />
-        return <ImageCluster imageElements={sample2} />
-    }
-
-    return <>
+    let text = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid enim dolore ipsa nihil blanditiis consectetur dolorem atque sint molestiae, ut dolores soluta perspiciatis temporibus quibusdam accusantium? Quo est dicta consequuntur. Voluptate laboriosam voluptas nam vel corporis? Cumque, aliquam ea, tempora, quos soluta eum esse delectus sunt quod perspiciatis rerum deserunt dolorum sint?"
+    let prompt = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid enim dolore ipsa nihil blanditiis consectetur dolorem atque sint molestiae, ut dolores soluta perspiciatis temporibus quibusdam accusantium? Quo est dicta consequuntur. Voluptate laboriosam voluptas nam vel corporis? Cumque, aliquam ea, tempora, quos soluta eum esse delectus sunt quod perspiciatis rerum deserunt dolorum sint?    (CLICK TO VIEW MORE)";
+    let text2 = "2 Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid enim dolore ipsa nihil blanditiis consectetur dolorem atque sint molestiae, ut dolores soluta perspiciatis temporibus quibusdam accusantium? Quo est dicta consequuntur. Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate laboriosam voluptas nam vel corporis? Cumque, aliquam ea, tempora, quos soluta eum esse delectus sunt quod perspiciatis rerum deserunt dolorum sint?";
+    let project= "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ipsum iaculis facilisi leo venenatis natoque vivamus ultricies faucibus sed. Leo sit tristique scelerisque cursus egestas aenean. ";
+    var project2= ["Gold Medal", "Best Educational Project"];
```

Comment:
always use `let`, there's never a reason to use var.

always use `const` wherever you can. Objects and arrays should always be initialized as `const`, unless you're overriding the entire array somewhere. You can make transactions to objects/array if they're defined as `const`.